packages/MsGraph/Actions/Event/GetEvents.cs

```diff
@@ -27,135 +27,116 @@ namespace Microsoft.Bot.Component.MsGraph.Actions.MSGraph
         /// </summary>
         /// <value><c>True</c> if we want to show only future events. <c>False</c> if otherwise.</value>
         [JsonProperty("futureEventsOnlyProperty")]
         public BoolExpression FutureEventsOnlyProperty { get; set; }
 
         /// <summary>
-        /// Gets or sets the max number of results to return from the query
+        /// Gets or sets the max number of results to return from the query.
         /// </summary>
-        /// <value>The max number of results</value>
+        /// <value>The max number of results.</value>
         [JsonProperty("maxResultsProperty")]
         public IntExpression MaxResultsProperty { get; set; }
 
+        /// <inheritdoc/>
         public override string DeclarativeType => GetEventsDeclarativeType;
 
-        /// <summary>
-        /// Calls the graph service to get a list of events based on the query
-        /// </summary>
-        /// <param name="client"></param>
-        /// <param name="dc"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        protected override async Task<List<CalendarSkillEventModel>> CallGraphServiceWithResultAsync(GraphServiceClient client, DialogContext dc, CancellationToken cancellationToken)
+        /// <inheritdoc/>
+        internal override async Task<List<CalendarSkillEventModel>> CallGraphServiceWithResultAsync(IGraphServiceClient client, IReadOnlyDictionary<string, object> parameters, CancellationToken cancellationToken)
         {
-            var dcState = dc.State;
-            var startProperty = this.StartProperty.GetValue(dcState);
-            var endProperty = this.EndProperty.GetValue(dcState);
-            var timeZoneProperty = this.TimeZoneProperty.GetValue(dcState);
-            var ordinalProperty = this.OrdinalProperty.GetValue(dcState);
-            var timeZone = TZConvert.GetTimeZoneInfo(timeZoneProperty);
-            var dateTimeTypeProperty = this.DateTimeTypeProperty.GetValue(dcState);
-            var isFuture = this.FutureEventsOnlyProperty.GetValue(dcState);
-            var maxResults = this.MaxResultsProperty.GetValue(dcState);
-            var userEmail = this.UserEmailProperty.GetValue(dcState);
-            var titleProperty = string.Empty;
-            var locationProperty = string.Empty;
-            var attendeesProperty = new List<string>();
-
-            if (this.TitleProperty != null)
-            {
-                titleProperty = this.TitleProperty.GetValue(dcState);
-            }
+            var startProperty = (DateTime?)parameters["StartProperty"];
+            var endProperty = (DateTime?)parameters["EndProperty"];
+            var timeZoneProperty = (string)parameters["TimeZoneProperty"];
+            var ordinalProperty = (OrdinalV2)parameters["OrdinalProperty"];
+            var dateTimeTypeProperty = (string)parameters["DateTimeTypeProperty"];
+            var isFuture = (bool)parameters["FutureEventsOnlyProperty"];
```

Comment:
This looks like the source of the test failure. The property name is different from the PopulateParameters step

Reply:
Yeah I just fixed that. Thanks!